XXX push upstream
Fix build race: WebKitDirectoryInputStreamData.h is generated by a custom
command owned by the WebKitShared OBJECT sub-target, but included from a source
compiled in the WebKitNetworkProcess sub-target, which has no dependency on it.

Index: Source/WebKit/PlatformGTK.cmake
--- Source/WebKit/PlatformGTK.cmake.orig
+++ Source/WebKit/PlatformGTK.cmake
@@ -124,6 +124,7 @@ add_custom_command(
     COMMAND ${PERL_EXECUTABLE} ${WEBCORE_DIR}/css/make-css-file-arrays.pl --defines "${FEATURE_DEFINES_WITH_SPACE_SEPARATOR}" ${WebKitGTK_DERIVED_SOURCES_DIR}/WebKitDirectoryInputStreamData.h ${WebKitGTK_DERIVED_SOURCES_DIR}/WebKitDirectoryInputStreamData.cpp ${WebKit_DirectoryInputStream_DATA}
     VERBATIM
 )
+list(APPEND WebKit_HEADERS ${WebKitGTK_DERIVED_SOURCES_DIR}/WebKitDirectoryInputStreamData.h)
 
 list(APPEND WebKit_PUBLIC_FRAMEWORK_HEADERS
     UIProcess/gtk/GtkVersioning.h
